vgall.go

```diff
@@ -0,0 +1,7 @@
+// +build !minimal
+
+package plot // import "gonum.org/v1/plot"
+
+import (
```

Comment:
I though we were just going to do the underscore imports here. Can you please do that please; an additional package just to do this seems overly complex.